src/Plugin/Field/FieldFormatter/StrawberryMetadataTwigFormatter.php

```diff
@@ -28,7 +28,7 @@
  *
  * @FieldFormatter(
  *   id = "strawberry_metadata_formatter",
- *   label = @Translation("Strawberry Field Metadata Formatter using Twig"),
+ *   label = @Translation("Strawberry Field Custom Metadata Template"),
```

Comment:
Just a minor thing here, can we keep the word formatter? Somewhere? make explaining/documenting easier IMHO

Reply:
Ok. But later I will ask you what does "Formatter" really mean to anyone! :)